lib/arena.rb

```diff
@@ -1,2 +1,49 @@
 class Arena
+
+  attr_accessor :name, :gladiators
+
+  def initialize(name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator(gladiator)
+    @gladiators << gladiator
+    # puts "adding gladiator #{gladiator}"
+  end
+
+  def add_gladiator(gladiator)
+    if @gladiators.length < 2
+      @gladiators << gladiator
+    end
+  end
+
+  def fight # no gladiators, does nothing
+  end
+
+  def start_fight(weapon)
+    if @gladiators.length == 2
+      #compare weapons (tridentVspear, clubVspear, tridentVclub, sameVsame)
+
+      #create if statements with weapon combinations
+      if @gladiators.weapon(0) && @gladiators.weapon(1)
```

Comment:
I expected to see come console.logs or a binding.pry here, to show that you are looking at the individual values.